ts/smelter-node/src/manager/existingInstance.ts

```diff
@@ -40,14 +44,35 @@ class ExistingInstanceManager implements SmelterManager {
   }
 
   public async setupInstance(opts: SetupInstanceOptions): Promise<void> {
-    // TODO: verify if options match
-    // https://github.com/software-mansion/smelter/issues/877
     await retry(async () => {
       await sleep(500);
-      return await this.sendRequest({
+      let status = (await this.sendRequest({
```

Comment:
the whole point for extracting the request to function is to use it where possible